master/static/srv/enrich_task_logs.py

```diff
@@ -130,13 +136,16 @@ def ship(self) -> None:
         tries = 0
         while tries < max_tries:
             try:
-                api.post(self.master_url, "task-logs", self.logs)
+                api.post(self.master_url, "task-logs", self.logs, cert=self.cert)
                 self.logs = []
                 return
-            except errors.APIException as e:
+            except Exception as e:
                 tries += 1
                 if tries == max_tries:
-                    raise e
+                    print(
+                        "failed to ship logs: {e}\nLogs to Ship: {self.logs}}",
```

Comment:
omit `self.logs`, maybe emit a `len(self.logs)` instead.  This log shipper still ships all logs to stdout naturally, so if somebody sees this message it'll just be really hard to read, but not share a lot of extra info.